helm/security-pack/templates/_helpers.tpl

```diff
@@ -15,43 +15,25 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+When apps are created in the org namespace add a cluster prefix.
+*/}}
+{{- define "app.name" -}}
+{{- if ne .cluster .ns -}}
+{{- printf "%s-%s" .cluster .app -}}
+{{- else -}}
+{{- .app -}}
+{{- end -}}
+{{- end -}}
+
 {{/*
 Common labels
 */}}
 {{- define "labels.common" -}}
-{{ include "labels.selector" . }}
 app-operator.giantswarm.io/version: 0.0.0
-app.giantswarm.io/branch: {{ .Chart.Annotations.branch | replace "#" "-" | replace "/" "-" | replace "." "-" | trunc 63 | trimSuffix "-" | quote }}
-app.giantswarm.io/commit: {{ .Chart.Annotations.commit | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
-app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 helm.sh/chart: {{ include "chart" . | quote }}
-giantswarm.io/service-type: {{ .Values.serviceType }}
+giantswarm.io/managed-by: {{ .Release.Name | quote }}
```

Comment:
I simplified the labels. Let me know if I cut too deeply 🔪 

I added the `managed-by label` as it means app-admission-controller skips some checks that are not relevant.
https://docs.giantswarm.io/app-platform/defaulting-validation/#gitops-support